computations.py

```diff
@@ -0,0 +1,140 @@
+import calendar
+import sys
+
+
+class Computations:
+
+    @staticmethod
+    def get_weather_attribute_index(weather_attributes_list, attribute):
+        return weather_attributes_list[attribute]
+
+    @staticmethod
+    def get_month_and_day_from_date(date):
+        year, month, day = date.split("-")
+        return int(month), int(day)
+
+    @staticmethod
+    def find_maximum_temperature_in_year(year, all_weather_records):
+        weather_attributes, weather_records = all_weather_records
+        maximum_temperature = Computations.get_weather_attribute_index(weather_attributes, "Max TemperatureC")
+        date = Computations.get_weather_attribute_index(weather_attributes, "PKT")
```

Comment:
In some files we also have `PKST`. Can you test different files and years?

Reply:
`date` still sound unclear!

Reply:
Yes we have PKST in some files. While Reading in Reader class we change PKST to PKT